File: Medium/Java/1442. Count Triplets That Can Form Two Arrays of Equal XOR.java
class Solution {
    public int countTriplets(int[] arr) {
        int count = 0;

        for(int i = 0; i < arr.length; i++){
            int temp = arr[i];
            for(int j = i + 1; j < arr.length; j++){
                // xor of element
                temp ^= arr[j];
                if(temp == 0)
                    count += j - i;
            }
        }

        return count;
    }
}

File: Medium/Python/1442. Count Triplets That Can Form Two Arrays of Equal XOR.py
class Solution(object):
    def countTriplets(self, arr):
        count = 0

        for i in range(len(arr)):
            temp = arr[i]
            for j in range(i + 1, len(arr)):
                # xor of element
                temp ^= arr[j]
                if temp == 0:
                    count += j - i

        return count
